### PR TITLE
added screens

### PR DESCRIPTION
**Pull request by Mandhularajitha**
added asreens

### CODE REVIEW

File: src/Components/CreatPost/CreatPost.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    font-weight: 500;
+  }
+  
+  .news_feed_title_content span {
+    color: #b7babe;
+    font-size: 12px;
+  }
+  
+  .news_feed_description {
+    min-height: 40px;
```
Comment (rahulmistry751) [inline src/Components/CreatPost/CreatPost.css:49]: use relative length units like rem

File: src/Components/CreatPost/CreatPost.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    align-items: center;
+  }
+  
+  .emojis img {
+    width: 25px;
+    margin-right: -3px;
+  }
+  
+  .emojis span {
+    color: #b7babe;
```
Comment (znjs) [inline src/Components/CreatPost/CreatPost.css:102]: can use CSS variables for colors

**Review by rahulmistry751 — APPROVED** [review]
Clean code and good folder structure

File: src/Components/EditComment/EditComment.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    color: #252b25;
+  }
+  
+  .share_upSide input:focus {
+    outline: none;
+  }
+  
+  .media_container hr {
+    border-style: solid;
+    /* border-color: rgba(224, 213, 213, 0.08); */
```
Comment (znjs) [inline src/Components/EditComment/EditComment.css:51]: can remove the commented code

File: src/Components/Navbar/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+::placeholder {
+  color: #a8abaf;
+}
+
+.active_icon {
+  border-bottom: 3px solid #2d88ff;
+}
+.active_icon i {
+  color: #2d88ff !important;
```
Comment (znjs) [inline src/Components/Navbar/Navbar.css:77]: avoid using `!important` if it's possible

File: src/Components/Profil/Profil.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  }
+  
+  .share_upSide input {
+    flex: 1;
+    padding: 60px 0 60px 60px;
+    background:  #e9e9f7;
+    border: none;
+    border-radius: 25px;
+    font-size: 16px;
+    color: #252b25;
```
Comment (znjs) [inline src/Components/Profil/Profil.css:39]: if possible use CSS variables

**Review by znjs — APPROVED** [review]
- Good PR
- Good naming conventions used

File: src/Components/CreatPost/CreatPost.css (hunk 2 above)
Comment (Mandhularajitha) [reply src/Components/CreatPost/CreatPost.css:102]: ok

File: src/Components/EditComment/EditComment.css (hunk 3 above)
Comment (Mandhularajitha) [reply src/Components/EditComment/EditComment.css:51]: ok

File: src/Components/Navbar/Navbar.css (hunk 4 above)
Comment (Mandhularajitha) [reply src/Components/Navbar/Navbar.css:77]: ok

**Pull request merged**